test/e2e/apimachinery/resource_quota.go

```diff
@@ -40,6 +40,7 @@ import (
 const (
 	// how long to wait for a resource quota update to occur
 	resourceQuotaTimeout = 30 * time.Second
+	podName              = "pfpod"
```

Comment:
is this needed?

Reply:
When I movedd both limit_range.go and resource_quote.go, the line is not needed is b/c resource_quota.go used `podName` which is defined in limit_range.go, and now they are separated in two folders, then it's needed.